core/src/databases/remote_databases/snowflake.rs

```diff
@@ -126,6 +162,15 @@ impl RemoteDatabase for SnowflakeRemoteDatabase {
             QueryDatabaseError::ExecutionError(anyhow!("Error creating session: {}", e).to_string())
         })?;
 
+        session
```

Comment:
unrelated, had to add this because any command that use `SELECT` needs to know which warehouse to run on.